lib/recurly/resource.rb

```diff
@@ -401,7 +407,12 @@ def from_xml xml
             }
             next
           end
-
+          
+          # Nokogiri on Jruby-1.7.19 likes to throw NullPointer exceptions
+          # if you try to run certian operations like el.attribute(''). Since
+          # we dont care about text nodes, let's just skip them
+          next if el.node_type == 3
```

Comment:
This seems like a magic number. Is there possibly a named constant in nokogiri we can reference?